GPU/Workflow/src/GPUWorkflowSpec.cxx

```diff
@@ -552,7 +552,8 @@ DataProcessorSpec getGPURecoWorkflowSpec(gpuworkflow::CompletionPolicyData* poli
               if (verbosity) {
```

Comment:
this file should not belong to your commit.

Reply:
clang format modified ot. So I ignore clang formats modification to files which are not related to me ?

Reply:
Hi @lietava : In principle you should use `git-clang-format [base branch]`. E.g., you rebase on origin dev, and then then run git-clang-format. This will then affect only files you have modified:
`git rebase origin/dev`
`git-clang-format origin/dev`
For this PR it is ok, no need to change it.